tarantool/connection.py

```diff
@@ -1269,6 +1291,19 @@ def flush_schema(self):
         self.schema.flush()
         self.load_schema()
 
+    def _schemafull_connection_check(self):
```

Comment:
It seems that `schemaful` is a more valid word (even though they both are not in dictionaries)
https://github.com/surrealdb/surrealdb/issues/115

Reply:
Done